src/main/java/org/folio/consortia/client/PermissionsClient.java

```diff
@@ -21,4 +22,7 @@ public interface PermissionsClient {
 
   @PostMapping(value = "/{userId}/permissions?indexField=userId", consumes = MediaType.APPLICATION_JSON_VALUE)
   void addPermission(@PathVariable("userId") String userId, Permission permission);
+
+  @DeleteMapping(value = "/perms/users/{id}")
+  void deleteUser(@PathVariable("id") String id);
```

Comment:
Better to rename to deletePermissionUser

Reply:
Changed